adapters/mobfoxpb/mobfoxpb.go

```diff
@@ -125,7 +155,5 @@ func getMediaTypeForImp(impID string, imps []openrtb.Imp) (openrtb_ext.BidType,
 	}
 
 	// This shouldnt happen. Lets handle it just incase by returning an error.
-	return "", &errortypes.BadServerResponse{
-		Message: fmt.Sprintf("Failed to find impression \"%s\" ", impID),
-	}
+	return "", nil
```

Comment:
Curious, why remove this error message?

Reply:
I can't find a way to test this keys, so looks like there is no way to really produce this error

Reply:
You can add a JSON test where the mock response from the bidder has a diff imp ID than what was in the request. That will trigger this case.  I'd suggest keeping that error message though so that you know if your users ever come across such an issue.